server/py/framework/db/sqldb/db.py

```diff
@@ -3131,12 +3126,16 @@ async def get_project_resources_counters(
             ),
         ) = results
         return (
-            project_to_files_count,
+            project_to_artifact_category_count[
+                mlrun.common.schemas.ArtifactCategories.other
+            ],
             project_to_schedule_count,
             project_to_schedule_pending_jobs_count,
             project_to_schedule_pending_workflows_count,
             project_to_feature_set_count,
-            project_to_models_count,
+            project_to_artifact_category_count[
```

Comment:
why datasets, models, and documents be under the same counter?

Reply:
That's a good question I don't have the answer for currently, but I asked @theSaarco the same, we decided not to change the current structure of the project summary.

Reply:
Also that's not exactly what we are counting:
models - > models
files -> any artifact that is NOT model/dataset/document - so `artifact` or `table` or `plot` etc.

Reply:
@quaark It’s worth adding a TODO to explain that